_data/osdf_namespace_metadata/envistor.yaml

```
@@ -11,14 +11,12 @@ description: |
 
   The EnviStor activity and underlying storage is funded through the [NSF Campus Cyberinfrastructure program](https://www.nsf.gov/funding/opportunities/cc-campus-cyberinfrastructure) under
   [Award # 2322308](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2322308).
-
 organization: Florida International University
 dataVisibility: public
-size: # In bytes so I can parse how I want
-bytesXferd: # In bytes so I can parse how I want
-url: https://www.cis.fiu.edu/kfscis-professor-awarded-500000-nsf-grant-for-environment/
+size: null # In bytes so I can parse how I want
+objectCount: null
 fieldOfScience: "Natural Resources and Conservation"
```

```suggestion
fieldOfScience: Natural Resources and Conservation
```
To ensure it matches the others

---

The quotations won't get parsed in the yaml, but I'll merge because why not